_posts/2022-12-07-solidity-developer-survey-2022-announcement.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+published: true
+title: 'Solidity Developer Survey 2022 is Live!'
+date: '2022-12-07'
+author: Franziska Heintel
+category: Announcements
+---
+
+It's that time of the year again: Today, we are launching the [Solidity Developer Survey 2022](https://cryptpad.fr/form/#/2/form/view/HuPIRv4gvziSV0dPV1SJncKzYJXTVc8LGCaMfLUoj2c/).
+
+Before we wrap up 2022 for good, we reach out to collect your feedback and insights!
+
+**📝 [TAKE THE SURVEY!](https://cryptpad.fr/form/#/2/form/view/HuPIRv4gvziSV0dPV1SJncKzYJXTVc8LGCaMfLUoj2c/) 📝**
+
+You can find the results of last year’s Solidity Developer Survey [here](https://blog.soliditylang.org/2022/02/07/solidity-developer-survey-2021-results/).
```

Comment:
```suggestion
You can find the previous results of the Solidity Developer Survey 2021 [here](https://blog.soliditylang.org/2022/02/07/solidity-developer-survey-2021-results/), where 435 people from 73 different countries participated, more than half of the respondents used Solidity for less than a year!
```

Added year explicitly for SEO.
Added hook for people to check out the previous blogpost.